test/util/util.go

```diff
@@ -284,6 +285,36 @@ func CreateJob(ctx context.Context, wds *kubernetes.Clientset, ns string, name s
 	}, timeout).Should(gomega.Succeed())
 }
 
+func GetDeploymentTime(ctx context.Context, clientset *kubernetes.Clientset, ns string, name string) time.Time {
+	var deployment *appsv1.Deployment
+	gomega.Eventually(func() error {
+		var err error
+		deployment, err = clientset.AppsV1().Deployments(ns).Get(ctx, name, metav1.GetOptions{})
+		return err
+	}, timeout).Should(gomega.Succeed())
+	return deployment.CreationTimestamp.Time
+}
+
+func GetBindingTime(ctx context.Context, clientset *ksClient.Clientset, name string) time.Time {
+	var binding *ksapi.Binding
+	gomega.Eventually(func() error {
+		var err error
+		binding, err = clientset.ControlV1alpha1().Bindings().Get(ctx, name, metav1.GetOptions{})
+		return err
+	}, timeout).Should(gomega.Succeed())
+	return binding.CreationTimestamp.Time
+}
+
+func GetManifestworkTime(ctx context.Context, ocmWorkImbs *ocmWorkClient.Clientset, ns string, name string) time.Time {
+	var manifestwork *v1.ManifestWork
+	gomega.Eventually(func() error {
+		var err error
+		manifestwork, err = ocmWorkImbs.WorkV1().ManifestWorks(ns).Get(ctx, name, metav1.GetOptions{})
+		return err
+	}, timeout).Should(gomega.Succeed())
+	return manifestwork.CreationTimestamp.Time
+}
+
```

Comment:
FYI, I think that you could replace these three funcs with one generic func. Can be pursued in follow-on work.

Reply:
BTW, I think that ginkgo has a built in templating (mainly for different testing) but maybe can be used here as well ..

Reply:
Those are good points.  I'll leave that for a follow up.